locale/sv/LC_MESSAGES/BingEarthWallpaper.po

```diff
@@ -0,0 +1,172 @@
+msgstr "Antal dagar innan bilder tas bort:"
+
+# There is no translations of "locale", in this context that I'm aware of. I would say that you use the english words in swedish aswell
+#: Settings.ui.h:9
+msgid "Bing locale:"
+msgstr ""
+
+#: Settings.ui.h:10
+msgid "Default is English - United States"
+msgstr "Standardspråk är engelska (USA)"
+
+#: Settings.ui.h:11
+msgid "Screen resolution:"
+msgstr "Skärmupplösning:"
+
+#: Settings.ui.h:12
+msgid "Override automatic resolution selection"
+msgstr "Välj manuell skärmupplösning"
+
+#: Settings.ui.h:13 extension.js:167
+msgid "Settings"
+msgstr "Inställningar"
+
+#: Settings.ui.h:14
+msgid "Gnome shell extension version: "
+msgstr "GNOME shell-tillägg version: "
+
+#: Settings.ui.h:15
+msgid ""
+"Changes your wallpaper daily to the Bing picture of the day for your region. "
+"Displays the picture description and copyright information."
+msgstr ""
+"Ändrar skrivbordsbakgrund till Bings dagliga bakgrundsbild för din region . "
+"Visar bild- och copyrightinformation."
+
+#: Settings.ui.h:16
+msgid "Maintained by Michael Carroll"
+msgstr "Underhålls av Michael Carroll"
+
+#: Settings.ui.h:17
+msgid "https://github.com/neffo/bing-wallpaper-gnome-extension"
+msgstr "https://github.com/neffo/bing-wallpaper-gnome-extension"
+
+#: Settings.ui.h:18
+msgid "Based on NASA APOD Gnome shell extension by Elia Argentieri"
+msgstr "Baserat på \"NASA APOD Gnome shell extensions\" av Elia Argentieri"
+
+#: Settings.ui.h:19
+msgid ""
+"<span size=\"small\">This program comes with ABSOLUTELY NO WARRANTY.\n"
+"See the <a href=\"https://www.gnu.org/licenses/old-licenses/gpl-2.0.html"
+"\">GNU General Public License, version 3 or later</a> for details.</span>"
+msgstr ""
+"<span size=\"small\">Detta program kommer INTE MED NÅGON GARANTI.\n"
+"Se <a href=\"https://www.gnu.org/licenses/old-licenses/gpl-2.0.html\">GNU "
+"General Public License, version 3 eller senare</a> för detaljer.</span>"
+
+#: Settings.ui.h:21
+msgid "About"
+msgstr "Om"
+
+#: extension.js:88 extension.js:91
+msgid "minutes"
+msgstr "minuter"
+
+#: extension.js:94
+msgid "days"
+msgstr "dagar"
+
+#: extension.js:97
+msgid "hours"
+msgstr "timmar"
+
+#: extension.js:152
+msgid "<No refresh scheduled>"
+msgstr "<Ingen uppdatering schemalagd>"
+
+#: extension.js:154 extension.js:157 extension.js:160
+msgid "Awaiting refresh..."
+msgstr "Uppdaterar..."
+
+#: extension.js:162
+msgid "Copy image to clipboard"
+msgstr "Kopiera bild till urklippet"
+
+#: extension.js:163
+msgid "Copy image URL to clipboard"
+msgstr "Kopiera bildlänk till urklippet"
+
+#: extension.js:166
+msgid "Refresh Now"
+msgstr "Uppdatera"
+
+#: extension.js:209
+msgid "Next refresh"
+msgstr "Nästa uppdatering"
+
+#: extension.js:394
+msgid "Bing Wallpaper of the Day for"
+msgstr "Bings dagliga foto"
+
+# I cant trigger the source text in the program so I don't know in what context it prompts. I guess it's when a picture can't be fetched from Bing? The translation could seem weird
```

Comment:
So these lines are triggered in the extension preferences screen when the user selects a market/locale that isn't provided by Bing in their region (for whatever reason).

![image](https://user-images.githubusercontent.com/5569112/88805335-5e251200-d1f2-11ea-8507-146c7a13787a.png)

In this case, I can't select the Slovenian locale in Australia.